src/routes/safe/components/Apps/components/ConfirmTxModal/ReviewConfirm.tsx

```diff
@@ -124,7 +124,8 @@ export const ReviewConfirm = ({
 
   const [buttonStatus, setButtonStatus] = useEstimationStatus(txEstimationExecutionStatus)
   const [executionApproved, setExecutionApproved] = useState<boolean>(true)
-  const doExecute = isExecution && executionApproved
+  const canTxExecute = useCanTxExecute()
```

Comment:
I would avoid renaming this and keep it as `isExecution`. It is named such in other files.

Reply:
I've renamed it intentionally so it becomes more self explanatory.

Reply:
Alright then. It does admittedly look a lot better when looking at it on the small difference pane.